addons/hr_holidays_attendance/tests/test_holidays_overtime.py

```diff
@@ -252,13 +252,26 @@ def test_leave_check_cancel(self):
             .action_cancel_leave()
         self.assertFalse(leave.overtime_id.exists())
 
-    def test_public_leave_overtime(self):
-        self.env['resource.calendar.leaves'].create([{
+    def test_multi_company_public_leave_overtime(self):
```

Comment:
I did not see it before (I thought you created a new test).

Why did you replace the existing test?

You should probably keep the existing test and add another one to check it in multi-company. 🤷🏼‍♂️

Reply:
It is more because I am not sure we could do that in stable.

I am not sure but I think in custom code, we could patch the test to disable it and so if the test no longer exists, I don't know if we have a traceback or not. 🤷🏼‍♂️ 
Or someone could extend the test to add more asserts. 🤷🏼‍♂️ 


I asked to @tivisse and he said we should just adapt the test without renaming it to avoid any issues in stable.

Reply:
okay, thanks for your time. i will fix it